delivery.yaml

```diff
@@ -1,26 +1,35 @@
-build_steps:
-  - desc: Prepare
-    cmd: |
-      curl -o /tmp/ensure-docker -f https://delivery.cloud.zalando.com/utils/ensure-docker
-      chmod +x /tmp/ensure-docker
-      /tmp/ensure-docker
+version: "2017-09-20"
 
-  - desc: Build
-    cmd: |
-      function npm {
-        docker run -i --rm -v "$(pwd):/workspace" -w /workspace registry.opensource.zalan.do/library/node-12.20-alpine:latest npm "$@"
-      }
+dependencies:
+  - id: node
+    type: docker
+    ref: registry.opensource.zalan.do/library/node-12.20-alpine
 
-      npm install --no-optional
-      npm run dist
+pipeline:
+  - id: build_and_publish
+    type: script
+    commands:
+      - desc: Prepare
+        cmd: |
+          curl -o /tmp/ensure-docker -f https://delivery.cloud.zalando.com/utils/ensure-docker
```

Comment:
I guess that's a very old leftover and can be thrown away, as docker is built into CDP build VMs since long ago )

Reply:
Good point removed it